tests/test_wog.py

```diff
@@ -12,6 +12,23 @@ def test_addled_grizzly():
 	assert wisp.atk == wisp.health == 3
 
 
+def test_blood_to_ichor():
+	game = prepare_game()
+	shieldbearer = game.player1.give("EX1_405").play()
+
+	def fire():
+		game.player1.give("OG_314").play(target=shieldbearer)
```

Comment:
Because the same line has to be called again outside of the loop, for the final BTI shot, and I have a powerful aversion to separated duplicate code. Would you prefer it that way? I would think this is the better practice, but I could also see how it may be overkill. The likelihood that something would need to be changed in the test is low, but if it did, you'd have to change it in fewer places. Your call.